default/scripting/buildings/shipyards/ORBITAL_DRYDOCK.focs.txt

```diff
@@ -15,6 +15,24 @@ BuildingType
     ]
     EnqueueLocation = [[ENQUEUE_BUILD_ONE_PER_PLANET]]
     effectsgroups = [
+        // With no pop, no repair or sitrep messages
+        EffectsGroup
+            scope = And [
+                Ship
+                InSystem id = Source.SystemID
+                OwnedBy empire = Source.Owner
+                Structure high = LocalCandidate.MaxStructure - 0.001
+            ]
+            activation = And [
+                Source
+                ContainedBy And [
+                    Planet
+                    Population high = 0
+                ]
+            ]
+            stackinggroup = "SHIP_REPAIR"
+            priority = 75
```

Comment:
How important is the priority for this to work? If the priority here being lower than those of the other effectsgroups is required for the no repair with no pop, then this is a potential source for troubles. If anyone decides to lower the values assigned to the priority macros, that will screw up the setup of the drydock effectsgroups.

Is it possible to calculate the priority? So we can do something like: `priority = [[VERY_EARLY_PRIORITY]] - 5`?